pkg/atomicbitops/atomic_bitops_common.go

```diff
@@ -145,3 +145,45 @@ func DecUnlessOneInt32(addr *int32) bool {
 		}
 	}
 }
+
```

Comment:
Is this change still necessary?  Won't the methods in atomic_bitops_noarch get used?

Reply:
Oh, yes; strangely, I expect this leads to a compile failure, like duplicated symbol. Anyway, I'll drop this in *_common.go.

FYI, when I do rebase to latest code, it seems that, we need to adapt vfs2 to this:

`pkg/sentry/vfs/file_description_impl_util.go:260:23: cannot use fd (type *FileDescription) as type memmap.MappingIdentity in assignment:
        *FileDescription does not implement memmap.MappingIdentity (missing AllowWrite method)`

Reply:
Another thing, when I do rebase to latest code, it seems that, we need to adapt vfs2 to this:

`pkg/sentry/vfs/file_description_impl_util.go:260:23: cannot use fd (type *FileDescription) as type memmap.MappingIdentity in assignment:
        *FileDescription does not implement memmap.MappingIdentity (missing AllowWrite method)`

So far, I tend to add a TODO for that; will support it for vfs2 until vfs2 becomes stable.